iati/home/templates/home/home_page.html

```diff
@@ -22,12 +22,12 @@
             </div>
             <div class="stat stat--flex">
                 <div class="stat__body stat__body--center">
-                    <h2 class="stat__heading stat__heading--large">{{ page.activities|intcomma }}</h2>
+                    <h2 class="stat__heading stat__heading--large" id="IATI-Website-Tests_num_iati_activities">{{ page.activities|intcomma }}</h2>
```

Comment:
Is prepending this with `IATI-Website-Tests` really necessary?

Reply:
Ah just seen the comment from @andylolz which makes a good point. In which case can it be a shorter prefix please?

Reply:
@andylolz suggested `smoketest_` but I prefer `IATI-Website-Tests_` since it makes it clear that the purpose of the `id` attribute is something related to IATI, thus reducing the risk that the ID field would be inadvertently deleted - which in turn would cause website tests to fail. 

The `IATI-Website-Tests` is also sufficient so that a google search of this term takes you straight to the repo with appropriate description on its purpose. Therefore I would argue keeping the ID name as is.

Reply:
Sounds good. Will update my review.